docs/index.html

```diff
@@ -26,7 +26,14 @@
 			function fixLinks() {
 				document.querySelectorAll("section a").forEach(
 					/** @param {HTMLAnchorElement} a */ a => {
-						if (a.host === location.host && a.pathname.includes("options.md")) {
+						const path = a.pathname;
```

Comment:
I wonder if we should switch this to spaces... tabs looks very weird... almost makes JS look like C++!

Reply:
(followup)

Reply:
Switch to spaces: 🤮 
Add `?ts=2` in URL to get 2-space indentation in your GitHub 😄

Reply:
🤺